ceaser_cipher.py

```
@@ -1 +1,33 @@
+alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+
+def caesar(start_text, shift_amount, cipher_direction):
+  end_text = ""
+  if cipher_direction == "decode":
+    shift_amount *= -1
+  for char in start_text:
+    if char in alphabet:
```

Nitpick: from the [UX](https://www.interaction-design.org/literature/topics/ux-design) perspective, should this be allowed? If a char is not supported (not in the alphabet), maybe it makes sense to notify the caller by throwing an error back at them instead of silently ignoring the problem? Or maybe it's not a problem at all? 😄